### PR TITLE
fixing code to pass checker

### PR DESCRIPTION
**Pull request by colinsauze**
This changes 

model.compile(#...
              loss='mse',
              #...)

to just

model.compile( loss='mse')

(and the same for the following example with the optimizer)

to make it pass the code check by using syntactically valid python. Is this acceptable or does it make it harder for a learner to understand?

We could also avoid this issue by not marking it as a python block of code or simply not presenting the full statement until we've covered both the loss and optimizer.

### CODE REVIEW

**Review by psteinb — APPROVED** [review]
LGTM! Thanks.
